css-logical-props/Overview.bs

```diff
@@ -22,7 +22,7 @@ Introduction</h1>
   This mapping controls the interpretation of logical keywords and properties.
 
 <h2 id="logical-controls">
-Logical Directional Values: ''block-start'', ''block-end'', ''inline-start'', ''inline-end''</h2>
+Logical Directional Values: 'block-start', 'block-end', 'inline-start', 'inline-end'</h2>
```

Comment:
These (and the next) are wrong - `'foo'` is the shorthand for property autolinks, `''foo''` is for values, and is being used correctly.